Automattic/wp-calypso
# Site Indictator: Remove calls to `recordGoogleEvent()`

Fixes https://github.com/Automattic/dotcom-forge/issues/3490

## Proposed Changes

Removes calls to `recordGoogleEvent()` from `<SiteIndicator />` and `<Site />`:

![CleanShot 2023-12-12 at 03 50 19@2x](https://github.com/Automattic/wp-calypso/assets/36432/bc1326d3-7850-478f-97c1-34a67f2484e2)

Google Analytics events are no longer in use: p1693487681855679-slack-C03N25JPCE4

## Testing Instructions

1. Interact with a `! isPreviewable` site (e.g. P2) and verify `calypso_mysites_sidebar_view_site_unpreviewable_clicked` still fires as expected.
2. Interact with a site that has a plugin update available and verify the link to `/wp-admin/update-core.php` still works as expected.
3. Interact with a site that has a broken Jetpack connection and verify `calypso_jetpack_site_indicator_disconnect_start` still fires when you start a disconnect.
